src/panel/poi/blocks/TimeTable.jsx

```diff
@@ -11,44 +12,55 @@ function showHour(day) {
   return _('Closed', 'hour block');
 }
 
-function showHours(displayHours) {
+const Days = ({ days }) => {
   const dayNumber = new Date().getDay();
 
-  return <tbody>
-    {displayHours.map((day, i) =>
-      <tr key={i} className={
-        classnames({ 'currentDay': (i + 1) % 7 === dayNumber })
-      }>
-        <td className="day">{ day.dayName }</td>
-        <td className="hours">{ showHour(day) }</td>
-      </tr>)}
-  </tbody>;
-}
+  return <table>
+    <tbody>
+      {days.map((day, i) =>
+        <tr key={i} className={
+          classnames({ 'currentDay': (i + 1) % 7 === dayNumber })
+        }>
+          <td className="day">{ day.dayName }</td>
+          <td className="hours">{ showHour(day) }</td>
+        </tr>)}
+    </tbody>
+  </table>;
+};
 
 const TimeTable = ({ title, schedule }) => {
-  // TODO: use OpeningHour instead (careful! OsmSchedule initialization happens there as well!)
-  if (schedule.isTwentyFourSeven) {
-    return <div
-      className="timetable timetable-status poi_panel__info__hours__24_7">
-      { _('Open 24/7', 'hour block') }
-      <div className="poi_panel__info__hour__circle"
-        style={{ background: schedule.status.color }}
-      />
-    </div>;
-  }
-
   const [ isCollapsed, setCollapsed ] = useState(true);
 
-  return <div className={classnames('timetable', { 'timetable--collapsed': isCollapsed })}>
-    <div className="timetable-status" onClick={() => { setCollapsed(!isCollapsed); }}>
-      <div className="timetable-status-text">{title}</div>
-      <i className="icon-icon_chevron-down" />
-    </div>
-    <div className={classnames('timetable-table')}>
-      <table>
-        { showHours(schedule.displayHours) }
-      </table>
+  let header;
+  let content;
+  if (title) {
+    header = title;
+    content = schedule.isTwentyFourSeven
+      ? <OpeningHour schedule={schedule} />
+      : <Days days={schedule.displayHours} />;
+  } else {
+    header = <OpeningHour schedule={schedule} />;
+    if (!schedule.isTwentyFourSeven) {
+      content = <Days days={schedule.displayHours} />;
+    }
+  }
+
+  const collapsable = !!content;
+  return <div className={classnames('timetable', {
+    'timetable--collapsable': collapsable,
+    'timetable--collapsed': isCollapsed,
+  })}>
+    <div className="timetable-status" onClick={() => {
+      if (collapsable) {
+        setCollapsed(!isCollapsed);
+      }
+    }}>
+      <div className="timetable-status-text">{header}</div>
```

Comment:
As a consequence, the default header ("open" or "closed") font-size has changed from 15px to 14px.

Reply:
I added an explicit rule for that https://github.com/QwantResearch/erdapfel/pull/590/commits/dc7c5ac9a6060e0a3fc3b73a961bae62de1b96a8
And I hope we'll have some time soon to clean the mess which is font-size hierarchy in the app :)